### PR TITLE
Add CLI examples for CloudWatch OAM

### PR DESCRIPTION
**Pull request by ashishbharadwajm**
*Issue #, if available:*

*Description of changes:*

Adding example CLI commands for Amazon CloudWatch Observability Access Manager: https://docs.aws.amazon.com/cli/latest/reference/oam/

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/examples/oam/delete-link.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+**To Delete a link**
+
+The following ``delete-link`` example deletes a link between a monitoring account sink and a source account. ::
+
+    aws oam delete-link \
+        --identifier arn:aws:oam:us-east-2:123456789111:link/a1b2c3d4-5678-90ab-cdef-example11111
+
```
Comment (elysahall) [inline awscli/examples/oam/delete-link.rst:7]: There is a missing output section. If this command produces no output please add the appropriate line.
https://aws.github.io/aws-cli/docs_styleguide.html#output

File: awscli/examples/oam/delete-sink.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+**To Delete a sink**
+
+The following ``delete-sink`` example deletes a sink. You must delete all links to a sink before you can delete that sink. ::
+
+    aws oam delete-sink \
+        --identifier arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345
+
```
Comment (elysahall) [inline awscli/examples/oam/delete-sink.rst:7]: Same here. There is a missing output section. If this command produces no output please add the appropriate line.
https://aws.github.io/aws-cli/docs_styleguide.html#output

File: awscli/examples/oam/get-link.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "Arn": "arn:aws:oam:us-east-2:123456789111:link/a1b2c3d4-5678-90ab-cdef-example11111",
+        "Id": "a1b2c3d4-5678-90ab-cdef-example11111",
+        "Label": "sourceAccount",
+        "LabelTemplate": "sourceAccount",
+        "ResourceTypes": [
+            "AWS::CloudWatch::Metric"
+        ],
+        "SinkArn": "arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345",
+        "Tags": {}
+}
```
Comment (elysahall) [inline awscli/examples/oam/get-link.rst:20]:
```suggestion
    }
```

File: awscli/examples/oam/get-sink-policy.rst [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    aws oam get-sink-policy \
+        --sink-identifier arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345
+
+Output::
+
+    {
+        "SinkArn": "arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345",
+        "SinkId": "a1b2c3d4-5678-90ab-cdef-example12345",
+        "Policy": "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"arn:aws:iam::123456789111:root\"},\"Action\":[\"oam:CreateLink\",\"oam:UpdateLink\"],\"Resource\":\"*\",\"Condition\":{\"ForAllValues:StringEquals\":{\"oam:ResourceTypes\":[\"AWS::Logs::LogGroup\",\"AWS::CloudWatch::Metric\",\"AWS::XRay::Trace\",\"AWS::ApplicationInsights::Application\"]}}}]}"
+}
```
Comment (elysahall) [inline awscli/examples/oam/get-sink-policy.rst:14]:
```suggestion
    }
```

File: awscli/examples/oam/put-sink-policy.rst [hunk 5]
```diff
@@ -0,0 +1,17 @@
+**To Creates or updates the resource policy**
+
+The following ``put-sink-policy`` example creates or updates the resource policy that grants permissions to source accounts to link to the monitoring account sink. ::
+
+    aws oam put-sink-policy \
+        --policy "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"arn:aws:iam::123456789111:root\"},\"Action\":[\"oam:CreateLink\",\"oam:UpdateLink\"],\"Resource\":\"*\",\"Condition\":{\"ForAllValues:StringEquals\":{\"oam:ResourceTypes\":[\"AWS::Logs::LogGroup\",\"AWS::CloudWatch::Metric\",\"AWS::XRay::Trace\",\"AWS::ApplicationInsights::Application\"]}}}]}" \
```
Comment (elysahall) [inline awscli/examples/oam/put-sink-policy.rst:6]:
```suggestion
        --policy '{"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":{"AWS":"arn:aws:iam::123456789111:root"},"Action":["oam:CreateLink","oam:UpdateLink"],"Resource":"*","Condition":{"ForAllValues:StringEquals":{"oam:ResourceTypes":["AWS::Logs::LogGroup","AWS::CloudWatch::Metric","AWS::XRay::Trace","AWS::ApplicationInsights::Application"]}}}]}' \
```

File: awscli/examples/oam/tag-resource.rst [hunk 6]
```diff
@@ -0,0 +1,9 @@
+**To Assign one or more tags to the specified resource**
+
+The following ``tag-resource`` example tags a sink ``arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345``. If the command succeeds, no output is returned. ::
+
+    aws oam tag-resource \
+        --resource-arn arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345 \
+        --tags team=Devops
```
Comment (elysahall) [inline awscli/examples/oam/tag-resource.rst:7]:
```suggestion
        --tags team=Devops

This command produces no output.
```

File: awscli/examples/oam/tag-resource.rst [hunk 7]
```diff
@@ -0,0 +1,9 @@
+**To Assign one or more tags to the specified resource**
+
+The following ``tag-resource`` example tags a sink ``arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345``. If the command succeeds, no output is returned. ::
```
Comment (elysahall) [inline awscli/examples/oam/tag-resource.rst:3]:
```suggestion
The following ``tag-resource`` example tags a sink ``arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345``. ::
```

File: awscli/examples/oam/untag-resource.rst [hunk 8]
```diff
@@ -0,0 +1,9 @@
+**To Removes one or more tags from the specified resource.**
+
+The following ``untag-resource`` example removes a tag with the key ``team`` from Sink ``arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345``. If the command succeeds, no output is returned. ::
+
+    aws oam untag-resource \
+        --resource-arn arn:aws:oam:us-east-2:123456789012:sink/f3f42f60-f0f2-425c-1234-12347bdd821f \
+        --tag-keys team
```
Comment (elysahall) [inline awscli/examples/oam/untag-resource.rst:7]:
```suggestion
        --tag-keys team

This command produces no output.
```

File: awscli/examples/oam/untag-resource.rst [hunk 9]
```diff
@@ -0,0 +1,9 @@
+**To Removes one or more tags from the specified resource.**
+
+The following ``untag-resource`` example removes a tag with the key ``team`` from Sink ``arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345``. If the command succeeds, no output is returned. ::
```
Comment (elysahall) [inline awscli/examples/oam/untag-resource.rst:3]:
```suggestion
The following ``untag-resource`` example removes a tag with the key ``team`` from Sink ``arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345``. ::
```

**Review by elysahall — CHANGES_REQUESTED** [review]

**Review by elysahall — CHANGES_REQUESTED** [review]
Missed a double colon causing incorrect rendering. Thanks!

File: awscli/examples/oam/untag-resource.rst [hunk 10]
```diff
@@ -0,0 +1,11 @@
+**To Remove one or more tags from the specified resource.**
+
+The following ``untag-resource`` example removes a tag with the key ``team`` from Sink ``arn:aws:oam:us-east-2:123456789012:sink/a1b2c3d4-5678-90ab-cdef-example12345``.
```
[the remark this answers is not in the document]
Comment (ashishbharadwajm) [reply awscli/examples/oam/untag-resource.rst]: Published the new commit with suggested changes

**Pull request closed without merging**